judo-runtime-core-jsl-itest/src/test/resources/model/mModelTC021.jsl

```diff
@@ -0,0 +1,16 @@
+model mModelTC021;
+
+import judo::types;
+
+type string sType(min-size = 0, max-size = 10);
+
+enum xEnum {
+    xA0 = 0;
+    xB1 = 1;
+}
+
+entity aEnt {
+    field Boolean fBool = true;
```

Comment:
These field name patterns won't work after JNG-4618.

Reply:
Of course. I am going to update it.